app/assets/json/translations/locale-en.json

```diff
@@ -1154,7 +1165,8 @@
             "leave_your_application": "Leave your application?",
             "listing_updated": "Listing Updated",
             "lottery_results_posted": "Lottery Results Posted",
-            "maximum": "maximum",
+            "maximum_income_month": "maximum income/month",
```

Comment:
Let's actually update these to use "per" instead of "/". That's easier to translate ("/"'s aren't super common in chinese, and then we have the added benefit of it making more sense for screenreaders